src/Action/Admin/RefundUnitsAction.php

```diff
@@ -46,6 +47,7 @@ public function __invoke(Request $request): Response
 
             $this->session->getFlashBag()->add('success', 'sylius_refund.units_successfully_refunded');
         } catch (CommandDispatchException $exception) {
+            Assert::notNull($exception->getPrevious());
```

Comment:
Using Assert::notNull() may result in throwing unhandled InvalidArgumentException - what about simple and more user friendly solution:

```
if (null === $exception->getPrevious()) {
    $this->session->getFlashBag()->add('error', 'sylius.ui. unexpected_error_occurred');
}
```

wdyt?

Reply:
The thing is, there will always be a _previous_ exception, as it's done automatically and silencing it could make more harm than good 😄 I'd leave it as it is (without it PHPStan is not passing)